src/display/components/table.rs

```diff
@@ -137,9 +136,9 @@ impl<'a> Table<'a> {
             vec![FIRST_COLUMN_WIDTHS[2], THIRD_COLUMN_WIDTHS[2]]
         } else {
             vec![
-                FIRST_COLUMN_WIDTHS[3],
-                SECOND_COLUMN_WIDTHS[0],
-                THIRD_COLUMN_WIDTHS[2],
+                rect.width * 53 / 100,
+                rect.width * 22 / 100,
+                rect.width * 23 / 100 - 1,
```

Comment:
I like this approach - do you think it will be possible to place the percentages in static consts for clarity (like the `FIRST_COLUMN_WIDTHS` etc? Maybe something like `MAX_FIRST_COLUMN_WIDTH_PERCENTAGE` or preferably a clearer shorter name if you can think of one? :)